src/ui/main_window.ui

```diff
@@ -289,16 +289,31 @@
    <property name="text">
     <string>MultiView</string>
    </property>
+   <property name="enabled">
+     <bool>false</bool>
+   </property>
+   <property name="checkable">
+     <bool>true</bool>
+   </property>
+   <property name="checked">
+     <bool>false</bool>
+   </property>
    <property name="toolTip">
-    <string>&lt;p&gt;Share the screen area between multiple open profiles.&lt;/p&gt;</string>
+    <string>&lt;p&gt;Enabled when multiple profiles are open, this splits the Mudlet screen to show them all at once.&lt;/p&gt;</string>
```

Comment:
> :bulb: Maybe move all menu tooltips to .cpp in the future to avoid HTML for translators..

I have considered that on a more general basis - then you end up with `dlgAboutDialog.cpp` :grinning: .

Actually though it isn't necessary - `lupdate` gathers identical texts from all sources  (and despite the use of HTML entities the texts are treated as the same) and presents them as a single text for translation - provided that the coders remember that there are multiple instances - of course the comments also have to be identical so individually numbering them (e.g. "x of y") in those comments stops that working - whereas ("ensure all y copies are kept the same") can alert others to the issue.

Reply:
Pretty sure that only works for strings in a same file though. 
So no dice for .ui files doubling .cpp texts like in this case.

Reply:
> Pretty sure that only works for strings in a same file though.

Nope - I'm pretty sure that it works for all files in the application - it is some of the black magic that `lupdate` does and why it is not easy to switch away from it - as it knows how to look in `.ui` files as well as `.h` and `.cpp` ones... - oh, looking at the `mudlet.ts` file I see it does separate the string sources by *classname* OTOH the Main window file might have been in the same class - if it had been called, say `mudlet.ui` rather than `main_window.ui` :man_shrugging:

Reply:
Ah, well, at least the translation memory system will spot identical source texts even in different files and suggest the translation of the first one in subsequent duplicate source texts in different files...